packaging/bundle-ebpf.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+SRCDIR="${1}"
+PLUGINDIR="${2}"
+
+EBPF_VERSION="$(cat "${SRCDIR}/packaging/ebpf.version")"
+EBPF_TARBALL="netdata-kernel-collector-glibc-${EBPF_VERSION}.tar.xz"
+
+mkdir -p "${SRCDIR}/tmp/ebpf"
```

Comment:
I have only one doubt for this package and also the dashboard,  would be better to use mktemp instead mkdir to create the directories?

Reply:
Normally I would agree, but because these are running in a build environment that gets recreated each time we try to build the packages, it's simpler to just use fixed paths here.